immich/upload.py

```
@@ -0,0 +1,364 @@
+                to_delete.append(_.filepath)
+
+    for filepath in to_delete:
+        main_deleted = True
+        if dry_run:
+            logger.info(f"Would have deleted {filepath}")
+        else:
+            try:
+                filepath.unlink()
+            except Exception as e:
+                main_deleted = False
+                logger.exception(f"Failed to delete {filepath}: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)